src/v/cluster/health_monitor_backend.cc

```diff
@@ -412,6 +421,17 @@ health_monitor_backend::get_cluster_health(
     co_return build_cluster_report(filter);
 }
 
+ss::future<storage::disk_space_alert>
+health_monitor_backend::get_cluster_disk_health(
+  force_refresh refresh, model::timeout_clock::time_point deadline) {
+    auto ec = co_await maybe_refresh_cluster_health(refresh, deadline);
+    if (ec) {
+        vlog(clusterlog.warn, "{} failed to refresh cluster health.", __func__);
```

Comment:
I'm just a `__func__y` dude, what can I say? 🤷   Couldn't resist, hehe. I will remove this bit as suggested.

Reply:
c++20 gives us https://en.cppreference.com/w/cpp/utility/source_location which we might consider integrating into `vlog` but it might end up making log lines unncessarily long